legal-api/src/legal_api/models/business.py

```diff
@@ -325,7 +331,7 @@ def json(self):
             'adminFreeze': self.admin_freeze or False,
             'state': self.state.name if self.state else Business.State.ACTIVE.name,
             'foundingDate': self.founding_date.isoformat(),
-            'goodStanding': self.good_standing,
+            'goodStanding': True if self.is_firm else self.good_standing,
```

Comment:
I think instead of doing this, you should update the `good_standing` function in the business model to return true if the business is a firm.

https://github.com/bcgov/lear/blob/296e726343398dd0db4723166ae22bda80461fc8/legal-api/src/legal_api/models/business.py#L289-L298